astropy/coordinates/angles/angle_lextab.py

```diff
@@ -5,13 +5,13 @@
 # remove it from this folder, then build astropy and run the tests in-place:
 #
 #   python setup.py build_ext --inplace
-#   pytest astropy/coordinates
+#   pytest astropy/coordinates/angles
```

Comment:
Just to be sure, you just rebuilt this, right?

Reply:
I believe so, but I'll do so again. I wrote this PR almost a year ago but never pushed it.

Reply:
✅ done.